frontend/src/components/logout/index.jsx

```diff
@@ -0,0 +1,41 @@
+import { h } from "preact";
+
+import { useLocation } from "preact-iso";
+import "./style.css";
+
+const Logout = () => {
+  const location = useLocation();
+  const handleLogout = async () => {
+    try {
+      const token = localStorage.getItem("jwt");
```

Comment:
Please use the jwt context like we do in the rest of the code:
```
const { jwt } = useContext(AuthContext);
```

Reply:
Done